src/jv.h

```diff
@@ -51,8 +52,13 @@ jv_kind jv_get_kind(jv);
 const char* jv_kind_name(jv_kind);
 static int jv_is_valid(jv x) { return jv_get_kind(x) != JV_KIND_INVALID; }
 
-jv jv_copy(jv);
-void jv_free(jv);
+void jv__free(jv);
+jv jv__copy(jv);
+
+static inline jv jv_copy(jv a){if(a.borrowed) return a; return jv__copy(a);}
+static inline void jv_free(jv a){if(!a.borrowed) jv__free(a);}
```

Comment:
Making those function static wrechs the api

Reply:
Making these `static inline` was my suggestion.  We can still export the original `jv_copy()`/`jv_free()` from libjq using a bit of pre-processor conditionals.

Reply:
We still need to address this.  When this PR is good shape I'll propose a patch.